### PR TITLE
Add multiple download URL support

### PR DESCRIPTION
**Pull request by dginovker**
Needs the json file hosted at a new path because otherwise people on old launchers would have issues.

I only tested with a local JSON and gtg right now, so if it breaks don't @ me, but it's a start 😄

### CODE REVIEW

File: Helper/GameFileTools.cs [hunk 1]
```diff
@@ -71,10 +71,11 @@ public static async Task<string> DownloadJSONFile()
             }
         }
 
-        public async Task DownloadFile(string pathtoZIPFile, string ZIPFileName, string DownloadUrl, IProgress<ProgressHelper> downloadProgress)
+        public async Task DownloadFile(string pathtoZIPFile, string ZIPFileName, string[] DownloadUrls, IProgress<ProgressHelper> downloadProgress)
         {
             try
             {
+                string DownloadUrl = DownloadUrls[new Random().Next(DownloadUrls.Length)];
```
Comment (dginovker) [inline Helper/GameFileTools.cs:78]: This is the real magic

**Comment by dginovker** [comment]
Updated for BFME 2 + RotWK support

Requires https://ravo92.github.io/GameFileDictionary2.json to exist (https://github.com/Ravo92/Ravo92.github.io/pull/1)

CICD is failing because PRs don't have permissions to create a release

File: BFME1/WinFormsMainGUI.cs [hunk 2]
```diff
@@ -540,7 +540,7 @@ private void PiBThemeSwitcher_Click(object sender, EventArgs e)
             }
         }
 
-        private async Task InstallUpdatRepairRoutine(string ZIPFileName, string DownloadUrl, string CorrectMD5HashValue)
+        private async Task InstallUpdatRepairRoutine(string ZIPFileName, string[] DownloadUrls, string CorrectMD5HashValue)
```
Comment (dginovker) [inline BFME1/WinFormsMainGUI.cs:543]: Btw this function should 100% be in the Helper class rather than slightly modified per game

Comment (Ravo92) [reply BFME1/WinFormsMainGUI.cs:543]: Youre correct!

**Pull request merged**